### DISCUSSION TITLE
Help with creating a framebuffer.

### DISCUSSION THREAD

**Discussion by David-Orangemoon** [greggman/twgl.js#232] [Q&A] [answered]
I've been trying to make a framebuffer using this bit of code.
```js
const triFrameBufferInfo = twgl.createFramebufferInfo(gl, [
    { format: gl.RGBA, type: gl.RGBA4, min: gl.LINEAR, wrap: gl.CLAMP_TO_EDGE },
    //{ format: gl.DEPTH_ATTACHMENT, },
  ],480,360);
```
I've been getting these errors too 
```log
WebGL warning: clear: Framebuffer not complete. (status: 0x8cd7) 
WebGL warning: clear: Framebuffer must be complete. 
```

**Comment by greggman** [comment, marked as answer]
It's not clear what you're trying to do but there's no such thing as `type: gl.RGBA4` which means you're getting `INVALID_ENUM` when the texture is created. Change it to `type: gl.UNSIGNED_BYTE`

Adding webgl-lint might help find these kinds of errors.

https://github.com/greggman/webgl-lint

**Reply by David-Orangemoon** [reply]
Hey thanks for the fast response. I've tried adding webgl lint to my project and yet I am still not getting any more useful information. Setting the type to unsigned byte seemed to do nothing either.
If there is any other info I could provide that could give more information I would be happy to do so. Thank you.

**Comment by David-Orangemoon** [comment]
I just figured out I was attaching it wrong lol. Sorry bout that
